src/components/Series/ScatterSeries.tsx

```diff
@@ -97,6 +100,7 @@ export function ScatterSeries<T extends SeriesPoint = SeriesPoint>(
     legendDispatch,
     otherProps.markerShape,
     otherProps.markerStyle?.fill,
+    props.markerShape,
```

Comment:
`otherProps.markerShape` is already here and contains the same value.

Reply:
fixed